### ISSUE TITLE
FSUM7332 syntax error: got ), expecting Newline

### ISSUE THREAD

**Issue by jthyssenrocket** [zowe/zowe-install-packaging#1727]
The issue described in https://github.com/zowe/zowe-install-packaging/issues/142 appears to be not resolved. 

I run Zowe as a STC through ZWESVSTC.
The user ID used for the STC does not have a home directory, hence does not have a .profile. 
Our system /etc/profile does not set autoconvert. 

The instance file is ISO8859-1 and tagged as text, but run-zowe.sh cannot execute it because auto convert is not enabled.

The STC will end with 
/xxxx/usr/lpp/zowe/inst1/instance.env 1: .: /xxxx/usr/lpp/zowe/inst1/bin/internal/read-instance.sh 4: .: /usr/lpp/zowe/inst1/bin/internal/run-zowe.sh 8: FSUM7332 syntax error: got ), expecting Newline

Either the instance.env file is incorrectly written as ISO8859-1 or read-instance.sh need to enable auto convert.

**Comment by 1000TurquoisePogs** [comment]
Which release did you see this issue with?

**Comment by jthyssenrocket** [comment]
1.15

**Comment by 1000TurquoisePogs** [comment]
I too noticed that instance.env is iso8859-1.
When the script to create an instance is run, it does not appear to explicitly do anything with encoding, but, it does set some environment variables (I thought mostly for nodejs use)
`export _CEE_RUNOPTS="FILETAG(AUTOCVT,AUTOTAG) POSIX(ON)"`
Reading between the lines, that may assume the default of new files to be 819 instead of 1047.
We need a 1047 bootstrapping process to occur.
It appears everything in /bin is already 1047, tagged as such
in ROOT_DIR/bin, everything is 1047 but untagged.

all of the tagged 1047 appears to be created by cat
while the instance.env in 819 is created by sed.
Not sure why this is different, but I guess it would be enough to run iconv & chtag after sed does its work

**Comment by jthyssenrocket** [comment]
Yes, converting and tagging instance.env should work, but I'd rather have it done automagically :) 

As a work around I added the following to the Zowe STC (which I've done for every release since Zowe 0.0 :)) 

```
//STDENV   DD *                                   
_BPXK_AUTOCVT=ON                                  
_CEE_RUNOPTS=FILETAG(AUTOCVT,AUTOTAG) POSIX(ON)   
```

**Comment by 1000TurquoisePogs** [comment]
Thats fine for now, but that is the fix I will attempt to implement. Seems best for it to be in 1047 rather than tagged ascii, so that it can be more easily edited from 3270.

**Comment by GilPen** [comment]
Hello,

I 'm trying to install 1.27.0 and I still have this error.

I coded the STDENV DD card as [jthyssenrocket] said but it doesn't correct the error.

Regards
Gilles Peniguel

**Comment by GilPen** [comment]
Hello

The complete error is : 
bin/validate.sh 1: .: validate_components 23: /usr/lpp/zowe/1.27.1/bin/internal/prepare-instance.sh 451: .: /usr/lpp/zowe/1.27.1/bin/internal/run-zowe.sh 42: FSUM7332 syntax error: got (, expecting Newline

The files in $ROOTDIR/bin/internal are untagged : 
£ ls -T                                    
- untagged    T=off HashFiles.class        
- untagged    T=off container-init.sh      
- untagged    T=off container-prestop.sh   
- untagged    T=off get-launch-components.s
- untagged    T=off prepare-environment.sh 
- untagged    T=off prepare-instance.sh    
- untagged    T=off run-zowe.sh            
- untagged    T=off start-component.sh     
- untagged    T=off zowe-set-env.sh      

Is it necessary to tag them in IBM-1047 code page ?

Regards
Gilles Peniguel

**Comment by MarkAckert** [comment]
As v1 is no longer maintained, I am closing this issue. If you encounter this in v2 or v3 you can re-open this issue. Thanks
